src/components/RecoverPass/index.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+//Import Material-UI
+import { Box, TextField, Button, Card, CardMedia, CardContent, Typography } from "@material-ui/core";
+//Import Formik
+import { useFormik } from "formik";
+//Import Styles
+import "./style.scss";
+//Import DATA
+import Logo from "assets/images/sushiMochi.jpg";
+
+//Import SchemaValidation
+import Schema from "./schema";
+
+const RecoverPass = () => {
+  const formik = useFormik({
+    initialValues: {
+      email: "",
+    },
+    validationSchema: Schema,
```

Comment:
```suggestion
    validationSchema: RecoverPassSchema,
```

Renombralo a algo un poco mas especifico, `Schema` es demasiado holgado. Te propongo que uses `NombreDelComponente` + `Schema` => quedaria `RecoverPassSchema`

Reply:
Done!